app/src/components/Day.js

```diff
@@ -48,38 +58,61 @@ const Day = ({ initialData }) => {
   const svg = dayData.svg;
 
   return (
-    <Card className="text-center" bg="dark" text="light">
-      <Card.Body>
-        <Card.Title>
-          <button onClick={() => changeDateValue(-1)}>
-            <i className="bi bi-arrow-left-circle"></i>
-          </button>{" "}
-          {date.format("ddd MMM D YYYY")}{" "}
-          <button onClick={() => changeDateValue(1)}>
-            <i className="bi bi-arrow-right-circle"></i>
-          </button>
-        </Card.Title>
-        <div
-          className="card-img-top mx-auto d-block"
-          dangerouslySetInnerHTML={{
-            __html: svg,
-          }}
-        ></div>
-        <Card.Text>
-          Moon Phase: {dayData.AlexPhase}
-          <br />
-          Action: {moonPhases[dayData.AlexPhase]?.action}
-          <br />
-          Details: {moonPhases[dayData.AlexPhase]?.desc}
-        </Card.Text>
-      </Card.Body>
-      <ListGroup className="list-group-flush">
-        <ListGroupItem variant="dark">
-          <AddTask loadTasks={loadTasks} date={date} />
-        </ListGroupItem>
-      </ListGroup>
-      <TaskList loadTasks={loadTasks} tasks={tasks} />
-    </Card>
+    <section>
+      <Card className="text-center" bg="dark" text="light">
+        <Card.Header>
+          Choose the Date:{" "}
+          <DatePicker
+            selected={date.toDate()}
+            onChange={(date) => {
+              setDate(dayjs(date));
+              history.push(`/day/${dayjs(date).format("YYYY-MM-DD")}`);
+            }}
+          />
+        </Card.Header>
+        <Card.Body>
+          <Card.Title>
+            <Link to={`/day/${date.add(-1, "day").format("YYYY-MM-DD")}`}>
+              <Button
+                variant="outline-info"
+                onClick={() => changeDateValue(-1)}
+              >
+                <i className="bi bi-arrow-left-circle"></i>
+              </Button>
+            </Link>{" "}
+            {date.format("ddd MMM D YYYY")}{" "}
+            <Link to={`/day/${date.add(1, "day").format("YYYY-MM-DD")}`}>
+              <Button variant="outline-info" onClick={() => changeDateValue(1)}>
+                <i className="bi bi-arrow-right-circle"></i>
+              </Button>
+            </Link>
+          </Card.Title>
+          <div
+            id="day-view-moon"
+            className="card-img-top mx-auto d-block"
+            dangerouslySetInnerHTML={{
+              __html: svg,
+            }}
```

Comment:
I bet if you made the SVG its own component, you wouldn't need `dangerouslySetInnerHTML` here.  A React component is basically a function anyway, so my advice above doesn't necessarily conflict with the advice in the previous sentence.